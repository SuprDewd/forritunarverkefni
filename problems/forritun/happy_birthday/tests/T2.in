Elsa
42
kvk
