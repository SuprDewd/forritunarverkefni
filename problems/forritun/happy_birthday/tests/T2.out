Hún á afmæli í dag,
hún á afmæli í dag.
Hún á afmæli hún Elsa,
hún á afmæli í dag.

Hún er 42 ára í dag,
hún er 42 ára í dag.
Hún er 42 ára hún Elsa,
hún er 42 ára í dag.
